public/src/Header/Header.js

```diff
@@ -0,0 +1,90 @@
+import React, {useState} from 'react';
+import {MdSearch, GiHamburgerMenu, BiCart, GrFormClose} from "react-icons/all";
+import logo from '../Z.png'
+
+
+function Header() {
+    const [search, setSearch] = useState("")
+    // Variables used in the header
+    const home = 'Home',
+        categories = 'Categories',
+        myAccount = 'My Account',
+        faq = 'F.A.Q.',
+        contact = 'Contact'
```

Comment:
We do have content system in the app now. Would it be to terribly annoying to try it out?

Reply:
Not at all, I'll get that switched over.